src/main/resources/resources/logisim/strings/fpga/fpga.properties

```diff
@@ -1,6 +1,7 @@
 FPGANotSupported = Not supported
 FPGASupported = Supported
 FPGAUnknown = Unknown
+FPGAHdlRequired = Required for HDL
```

Comment:
I am not quite sure about the context of this string, which makes it difficult to judge how well it reflects the intention.

Reply:
This was hardcoded string in the attribute cell renderer. It is used when you add the component that is HDL supported (i.e. if you add existing circuit to you current one (pic rel)) and you have empty label. It will be shown instead of label unless you correct that. I do not like that message either, yet got not better alternative.

![ss](https://user-images.githubusercontent.com/8041294/200268578-4a03b7ee-e934-46b1-b925-72147164803d.png)